### PR TITLE
CTAA-1385 Enabled SSL pinning for the CDN, add debug button to trigger download

### PR DESCRIPTION
**Pull request by finnsch**
## Changes

* Added SSL pinning for the CDN API
* Added production endpoint for the CDN API
* Added two debug buttons to trigger download of all batches (full and daily batches) and only the full batch

### CODE REVIEW

File: CoronaContact/AppConfiguration.swift [hunk 1]
```diff
@@ -28,4 +28,8 @@ enum AppConfiguration: ConfigurationRetrievable {
     static var apiSmsHostName: String {
         AppConfiguration.value(for: "API_SMS_HOST")
     }
+
+    static var apiCdnHostName: String {
```
Comment (nilsgrabenhorst) [inline CoronaContact/AppConfiguration.swift:32]: Apple recommends to capitalise acronyms when naming things. One could argue that also applies to "API" but that would look weird. How about `apiCDNHostName`?

**Review by nilsgrabenhorst — APPROVED** [review]
Just a small naming suggestion.

File: CoronaContact/AppConfiguration.swift (hunk 1 above)
Comment (finnsch) [reply CoronaContact/AppConfiguration.swift:32]: I agree with you but then I would have to rename the `apiSmsHostName` property as well. I would rather not do it with this pull request.

Comment (nilsgrabenhorst) [reply CoronaContact/AppConfiguration.swift:32]: Fair enough

**Pull request merged**